netci.groovy

```diff
@@ -297,6 +297,41 @@ def buildArchConfiguration = ['Debug': 'x86',
 }
 
 // **************************
+// Define uap and uapaot vertical builds that will run on every merge.
+// **************************
+[true, false].each { isPR ->
+    ['uap', 'uapaot'].each { targetGroup ->
+        ['Debug'].each { configurationGroup ->
+            ['Windows_NT'].each { osName ->
+                def osGroup = osGroupMap[osName]
+                def osForMachineAffinity = osName
+
+                def newJobName = "${targetGroup}_${configurationGroup.toLowerCase()}"
+
+                def newJob = job(Utilities.getFullJobName(project, newJobName, isPR)) {
+                    // On Windows we use the packer to put together everything. On *nix we use tar
+                    steps {
+                        batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" x86 && build.cmd -${configurationGroup} -os:${osGroup} -buildArch:${buildArchConfiguration[configurationGroup]} -framework:${targetGroup}")
```

Comment:
nope, I noticed that they will be x86 once the jobs where generated. That should still work so I have already kicked off a couple of runs, but I will change them back to x64 before merging.